src/default.nix

```
@@ -341,6 +342,7 @@
 
   makeOutputsForDreamLock = {
     dreamLock,
+    sourceRoot,
```

This should be optional.

```suggestion
    sourceRoot ? null,
```